tests/test_mlip_calculators.py

```diff
@@ -2,63 +2,137 @@
+        ("chgnet", "cpu", {"model": "0.2.0"}),
+        ("chgnet", "cpu", {"model_path": CHGNET_PATH}),
+        ("chgnet", "cpu", {"model": CHGNET_MODEL}),
+    ],
+)
 def test_mlips(architecture, device, kwargs):
     """Test mace calculators can be configured."""
     calculator = choose_calculator(architecture=architecture, device=device, **kwargs)
     assert calculator.parameters["version"] is not None
 
 
-test_extra_mlips_data = [("alignn", "cpu")]
-
-
-@pytest.mark.extra_mlips
-@pytest.mark.parametrize("architecture, device", test_extra_mlips_data)
-def test_extra_mlips(architecture, device):
-    """Test m3gnet and chgnet calculators can be configured."""
-    calculator = choose_calculator(
-        architecture=architecture,
-        device=device,
-    )
-    assert calculator.parameters["version"] is not None
-
-
 def test_invalid_arch():
     """Test error raised for invalid architecture."""
     with pytest.raises(ValueError):
         choose_calculator(architecture="invalid")
 
 
-def test_model_model_paths():
-    """Test error raised if both model and model_paths are specified."""
+@pytest.mark.parametrize(
+    "architecture, model_path",
+    [
+        ("mace", "/invalid/path"),
+        ("mace_off", "/invalid/path"),
+        ("mace_mp", "/invalid/path"),
+        ("m3gnet", "/invalid/path"),
+        ("chgnet", "/invalid/path"),
+    ],
+)
+def test_invalid_model_path(architecture, model_path):
+    """Test error raised for invalid model_path."""
+    with pytest.raises((ValueError, RuntimeError)):
+        choose_calculator(architecture=architecture, model_path=model_path)
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"model": MACE_MP_PATH, "model_paths": MACE_MP_PATH},
+        {"model_path": MACE_MP_PATH, "model_paths": MACE_MP_PATH},
+        {"model_path": MACE_MP_PATH, "model": MACE_MP_PATH},
+        {"model_path": MACE_MP_PATH, "potential": MACE_MP_PATH},
+    ],
+)
+def test_model_model_paths(kwargs):
+    """Test error raised if model path is specified in multiple ways."""
     with pytest.raises(ValueError):
-        choose_calculator(
-            architecture="mace",
-            model=MODEL_PATH,
-            model_paths=MODEL_PATH,
-        )
+        choose_calculator(architecture="mace", **kwargs)
 
 
 @pytest.mark.parametrize("architecture", ["mace_mp", "mace_off"])
 def test_invalid_device(architecture):
     """Test error raised for invalid device is specified."""
     with pytest.raises(ValueError):
         choose_calculator(architecture=architecture, device="invalid")
+
+
+@pytest.mark.extra_mlips
+@pytest.mark.parametrize(
+    "architecture, device, kwargs",
+    [
+        ("alignn", "cpu", {}),
+        ("alignn", "cpu", {"model_path": {MODEL_PATH} / "v5.27.2024"}),
+        ("alignn", "cpu", {"model_path": {MODEL_PATH} / "v5.27.2024/best_model.pt"}),
+        ("alignn", "cpu", {"model": "alignnff_wt10"}),
+        ("alignn", "cpu", {"path": {MODEL_PATH} / "v5.27.2024"}),
```

Comment:
Do you mean the curly braces here? Won't that create a `set`?

Reply:
Well spotted, thanks! I initially thought it was going to be a f string, and didn't clean up properly